spk/maraschino-custom/src/wizard/install_uifile_enu

```diff
@@ -0,0 +1,22 @@
+[{
+    "step_title": "Choose your fork",
+    "items": [{
+        "type": "textfield",
+        "desc": "Configure the fork to use",
+        "subitems": [{
+            "key": "wizard_fork_url",
+            "desc": "Fork URL",
+            "defaultValue": "git://github.com/gugahoi/maraschino.git",
```

Comment:
Seeing as some forks disappear after a while or are left unmaintained (we've seen that with the other custom packages), maybe just use the original Maraschino link here?